Tema4/src/Local_Store.java

```diff
@@ -0,0 +1,33 @@
+import java.time.LocalDate;
+
+public class Local_Store {
```

Comment:
this is an adaptation of what we did during class... but not entirely what the homework was about.